web-registry/src/pages/Dashboard/MyEcocredits/hooks/useCreateSellOrderSubmit.tsx

```diff
@@ -67,7 +68,17 @@ const useCreateSellOrderSubmit = ({
       if (batchDenom && amount && askDenom) {
         const baseDenom = await getDenomtrace({ denom: askDenom });
 
+        const projectId = batchDenom.substring(0, batchDenom.indexOf('-', 4));
+        const projectName = await getProjectName(projectId);
```

Comment:
Not sure why we need to make a request here, the project name has already been retrieved at this point since it is displayed in the credits table.
I think we just have to pass `sellOrderCreateOpen` an `credits` from above to avoid the request here.

Reply:
good point, thanks!